heareval/tasks/pipeline.py

```diff
@@ -801,18 +803,23 @@ def run(self):
         shutil.copytree(self.requires()["resample"].workdir, self.workdir)
 
         # Copy the traintestmetadata
-        shutil.copytree(
+        # and vocabmetadata
+        for src in [
             self.requires()["traintestmeta"].workdir,
-            self.workdir,
-            dirs_exist_ok=True,
-            ignore=shutil.ignore_patterns("*.csv"),
-        )
-        # Copy the vocabmetadata
-        shutil.copytree(
             self.requires()["vocabmeta"].workdir,
-            self.workdir,
-            dirs_exist_ok=True,
-        )
+        ]:
+            dst = self.workdir
+            for item in os.listdir(src):
+                if item.endswith(".csv"):
+                    continue
```

Comment:
We have to avoid copying the .csv only in the traintestmeta as we are using the json versions from this folder. For vocabmeta folder we need the .csv files as well.

Reply:
Fixed this.